Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM cosmtrek/air:latest
+
+RUN go install github.com/go-delve/delve/cmd/dlv@latest
+
+ENTRYPOINT [ "air" ]
```

Comment:
Normally the `Dockerfile` file at the root has something to do with the underlying codebase. Since, this is just the `air` image (with `delve` installation), I'd rather put it in the `local/jimm` directory.

Also, a comment might be helpful on why we use this image. Maybe just comment before line 3, saying *"Installing delve in the image to avoid running `go install` in running containers"*.